src/sentry/tasks/sentry_apps.py

```diff
@@ -449,6 +473,7 @@ def send_webhooks(installation, event, **kwargs):
         kwargs["install"] = installation
 
         request_data = AppPlatformEvent(**kwargs)
+        assert servicehook.sentry_app, "sentry app must exist to get webhook url"
```

Comment:
Interesting that we use both `installation.sentry_app` and `servicehook.sentry_app` here 🤔

Reply:
🌵